imsc1/ttml/region/region-timing.ttml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tt ttp:cellResolution="60 20" xml:lang="en"
+	xmlns="http://www.w3.org/ns/ttml"
+	xmlns:ttp="http://www.w3.org/ns/ttml#parameter" 
+	xmlns:tts="http://www.w3.org/ns/ttml#styling">
+  <head>
+    <layout>
+      <region xml:id="r1" begin="0s" end="10s" tts:backgroundColor="red" tts:color="white" tts:origin="5% 5%" tts:extent="80% 20%" tts:displayAlign="after" tts:showBackground="whenActive"/>
+      <region xml:id="r2" begin="10s" end="20s" tts:backgroundColor="green" tts:color="white" tts:origin="5% 25%" tts:extent="80% 40%" tts:displayAlign="after" tts:showBackground="whenActive"/>
+      <region xml:id="r3" begin="20s" end="30s" tts:backgroundColor="blue" tts:color="white" tts:origin="5% 65%" tts:extent="80% 20%" tts:displayAlign="after" tts:showBackground="always"/>
+    </layout>
+  </head>
+ <body>
+   <div region="r1">
+     <p>This text should only appear during the interval [0s,10s]</p>
```

Comment:
TTML interval timings are exclusive of the end time, which is conventionally written `[0s,10s)`, i.e. the closing bracket is a round bracket rather than a square one.